gtsfm/utils/io.py

```diff
@@ -301,15 +302,19 @@ def write_cameras(gtsfm_data: GtsfmData, images: List[Image], save_dir: str) ->
             calibration = camera.calibration()
 
             fx = calibration.fx()
-            u0 = calibration.px()
-            v0 = calibration.py()
+            fy = calibration.fy()
+            u0 = calibration.px()  # optical image center
+            v0 = calibration.py()  # optical image center
             k1 = calibration.k1()
             k2 = calibration.k2()
+            p1 = 0.0
+            p2 = 0.0
+            # missing tangential distortion coefficients
 
             image_height = images[i].height
             image_width = images[i].width
 
-            f.write(f"{i} {camera_model} {image_width} {image_height} {fx} {u0} {v0} {k1} {k2}\n")
+            f.write(f"{i} {camera_model} {image_width} {image_height} {fx} {fy} {u0} {v0} {k1} {k2} {p1} {p2}\n")
```

Comment:
same note here -- this doesn't correspond to the colmap "Radial" format afaik

https://github.com/colmap/colmap/blob/5c1d58a085920c41cf2d9c892da8041b1fc0c86d/src/colmap/sensor/models.h#L225-L231

Reply:
(these are OPENCV models)

https://github.com/colmap/colmap/blob/5c1d58a085920c41cf2d9c892da8041b1fc0c86d/src/colmap/sensor/models.h#L239-L247